nix/eval-modules.nix

```diff
@@ -0,0 +1,15 @@
+rec {
+  evalModules = { pkgs, name, modules }: (pkgs.lib.evalModules {
```

Comment:
Could you add document the function as a comment above? Here, and in `makeProcessCompose` function below.

Reply:
Done.